ethereum/bindings/Cargo.toml

```diff
@@ -7,4 +7,3 @@ edition = "2021"
 
 [dependencies]
 ethers = { version = "2", default-features = false, features = ["abigen"] }
-serde = "1"
```

Comment:
No, this is a bug... it will not compile if a regeneration is forced. `serde` is necessary.